updating/updating-cluster-cli.adoc

```diff
@@ -25,6 +25,7 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being updated to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[Upgrading an OpenShift Container Platform cluster configured for manual mode with STS].
 * Ensure that you address all `Upgradeable=False` conditions so the cluster allows an update to the next minor version. An alert displays at the top of the *Cluster Settings* page when you have one or more cluster Operators that cannot be upgraded. You can still update to the next available patch update for the minor release you are currently on.
 * Review the list of APIs that were removed in Kubernetes 1.25, migrate any affected components to use the new API version, and provide the administrator acknowledgment. For more information, see xref:../updating/updating-cluster-prepare.adoc#updating-cluster-prepare[Preparing to update to {product-title} 4.12].
+* If you run an Operator or you have configured any application with the Pod Disruption Budget, you might experience a block during the upgrade process. The `PodDisruptionBudget` field with the `minAvailable: 1` value may block the eviction process while nodes are drained to apply pending MachineConfigs. If several nodes are rebooted, all the pods may run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
```

Comment:
`If several nodes are rebooted, all the pods may run on only one node, and the ``PodDisruptionBudget`` field can prevent the node drain.` 
Does this mean:
`If several nodes are rebooted, because all the pods might run on only one node, the ``PodDisruptionBudget`` field can prevent the node drain.` ?
If yes, you can use this statement.

Reply:
I think adding because will change the meaning; hence keeping the sentence as is